lib/fastly_nsq/messenger.rb

```diff
@@ -4,12 +4,12 @@ module FastlyNsq::Messenger
 
   module_function
 
-  def deliver(message:, on_topic:, originating_service: nil)
+  def deliver(message:, on_topic:, originating_service: nil, meta: {})
     payload = {
       data: message,
       meta: {
         originating_service: originating_service || self.originating_service,
-      },
+      }.merge(meta),
```

Comment:
don't you want this to merge into the existing `meta` key?

Reply:
oh man. that diff confused me.

Reply:
hah yea that does look weird in the diff, but it is doing the "right" thing.